dockerfiles/ci/xfail_tests/README.md

```diff
@@ -2,6 +2,21 @@
 
 This file explains why we decided to disable specific PHP language tests. Investigations for tests disabled before this file was created are not present.
 
+## `ext/pcntl/tests/pcntl_unshare_01.phpt`
+
+Disabled on versions: `7.4` (it wasn't there on [7.3-](https://github.com/php/php-src/tree/PHP-7.3/ext/pcntl/tests)).
+
+Links to sample broken executions: [7.4](https://app.circleci.com/pipelines/github/DataDog/dd-trace-php/5532/workflows/2d26f68b-fb78-46f1-b846-c9e0f1c5cefc/jobs/377363).
+
+_Investigation_
+
+`unshare` requires processes [not to be threaded](https://man7.org/linux/man-pages/man2/unshare.2.html) to use the flag `CLONE_NEWUSER`. In our case the background sender is started in a thread and causes the `CLONE_NEWUSER` to be flagged as an error.
```

Comment:
How do you recommend we handle this case for users who have `pcntl_unshare(CLONE_NEWUSER);` somewhere in their application? Do we hook the `pcntl_unshare` function handler and make it noop and leave a debug message explaining that `CLONE_NEWUSER` is not supported when `ext/ddtrace` is enabled?

Reply:
Good point. I don't feel comfortable with noop-ing `pcntl_unshare` since changing the execution pattern we can cause security/billing/<whatever> issues, or damage in general.

I added a note the our release docs! But this is one of those cases when it is better to fail for good. https://github.com/DataDog/documentation/pull/10407